lib/errors.js

```diff
@@ -32,3 +32,25 @@ AuthError.__proto__ = Error;
 AuthError.prototype.__proto__ = Error.prototype;
 
 exports['AuthError'] = AuthError;
+
+
+// argument: { backupPath: '...', configuration: { ... } }
+function ClientResetError(argument) {
+    const error = Error.call(this, 'ClientResetError');
+
+    this.name = 'ClientResetError';
+    this.message = error.message;
+    this.stack = error.stack;
+
+    this.getBackupPath = function() {
```

Comment:
Why not expose a configuration here like we do for "offline" client reset? That's by the far the easiest way for a client to access the backed-up Realm.